src/plugins/internal/InternalInstallPlugin.ml

```diff
@@ -146,9 +77,12 @@ let install pkg argv =
       fun fn -> fn
   in
 
-  let install_file ?tgt_fn src_file envdir =
+  let install_file ?(with_destdir=true) ?tgt_fn src_file envdir =
```

Comment:
`ocamlfind install` is, ultimately, just a copy in the right directory. ocamlfind is still used to figure out where to copy, so I don't see the problem. I considered doing method 2, so I could change it to that if you prefer it. But I don't think maintaining two code path to do two similar task is a great idea.